IntelliTect.TestTools.Console/NormalizeOptions.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace IntelliTect.TestTools.Console
+{
+    /// <summary>
+    /// Provides available options for normalizing expected input and output
+    /// </summary>
+    [Flags]
+    public enum NormalizeOptions
+    {
+        /// <summary>
+        /// No normalization.
+        /// </summary>
+        None = 0,
+        
+        /// <summary>
+        /// Whether differences in line ending styles should be ignored.
+        /// </summary>
+        NormalizeLineEndings = 1,
+        
+        /// <summary>
+        /// Whether ansi color code characters should be ignored.
+        /// </summary>
+        StripAnsiEscapeCodes = 2,
```

Comment:
How about a `Default` value set to `StripAnsiEscapeCodes | NormalizeLineEndings`?